website/src/components/Header/Header.tsx

```diff
@@ -77,7 +83,7 @@ export const Header = () => {
             <Logo className="h-[36px]" />
           </Link>
           <div className="flex flex-row items-center gap-6 md:order-2 space-x-3 md:space-x-0 rtl:space-x-reverse grow justify-end basis-0">
-            <Link href={MLFLOW_DOCS_URL} className="hidden md:block">
+            <Link href={getStartedHref} className="hidden md:block">
```

Comment:
does this suffice? same for below. then we don't need lines 36-39

```suggestion
            <Link href="#get-started" className="hidden md:block">
```